src/components/ArticleItem.jsx

```diff
@@ -0,0 +1,29 @@
+import { Link } from "react-router-dom";
+import formatDate from "../utilities/formatDate";
+
+export default function ArticleItem({ article }) {
+  return (
+    <li className="flex max-w-[500px] flex-col gap-2 rounded-xl bg-gray-200 p-4 shadow-xl md:max-w-[400px]">
+      <h3 className="font-black">
+        <Link className="underline" to={`/articles/${article.article_id}`}>
+          {article.title}
+        </Link>
+      </h3>
+      <p>
+        <span>By </span>
+        <Link
+          className="text-blue-500 underline"
+          to={`/users/${article.author}`}
+        >
+          {article.author}
+        </Link>
+      </p>
+      <p className="text-sm text-gray-500">{formatDate(article.created_at)}</p>
+      <img src={article.article_img_url} alt="" className="my-2 rounded-xl" />
```

Comment:
This NEEDS an alt tag. And make it as accurate as possible. Remember that accessibility is a huge deal and needs to be considered throughout this project and beyond!